src/expertsystem/reaction/solving.py

```diff
@@ -148,15 +163,13 @@ def get_name(rule: Any) -> str:
     return converted_dict
 
 
-class Result:
+class QNResult:
```

Comment:
The name is not that instructive. But I think eventually (as discussed elsewhere) this class needs to be split into a container of solutions and some bookkeeping clss for violated rules etc.

Reply:
I can also rename those to `QuantumNumberResult` result.